docs/src/setup/configuration.md

```diff
@@ -44,6 +44,22 @@ Please refer to the [Canboat PGN database](https://canboat.github.io/canboat/can
 
 **_NMEA0183_**: The processing of NMEA0183 sentences is done by [nmea0183-signalk](https://github.com/SignalK/signalk-parser-nmea0183)
 
+#### NMEA 0183 Options
+
+- _Suppress nmea0183out event_ - All incoming NMEA0183 data is made available over TCP on port 10110 by default. This happens by incoming data emitting _nmea0183out_ event. Suppressing the event will prevent data from this connection to being routed to TCP.
+
+- _Input Event_ - All incoming data is being emitted as _nmea0183_ and _nmea0183out_ events by default, but with these events you can not distinguish between input from multiple NMEA 0183 connections. By adding your own input event you can process data from this connection specifically.
```

Comment:
By default, data received on this connection will cause the _nmea0183_ and _nmea0183out_ events to be emitted. In order to distinguish input from this connection from other NMEA 0183 connections, enter an input event name which will be emitted _(in addition to the default devents)_ when data is received on this connection.